### ISSUE TITLE
Revision – 630 Total Views of Asynchronous Program Presentations within 30 Days (TOTVIEWS)

### ISSUE THREAD

**Issue by mariabernier10601** [IMLS/public-libraries-survey#63]
**Name:**
Maria Bernier

**State/Affiliation:**
SDC, Connecticut State Library 

**Description of Change:**
_Existing Data Element:_

- 630 Total Views of Asynchronous Program Presentations within 30 Days (TOTVIEWS)

_Current Definition:_ The count of views of asynchronous program presentations for a period of thirty (30) days after the presentation was posted, even if that period extends beyond the survey reporting period (or fiscal year). For program presentations made available via Facebook, count unique 1-minute views of each video. For those made available via other platforms, count unique views of each video. 

_Revised Definition:_ The count of views OR PLAYS of asynchronous program presentations for a period of thirty (30) days after the presentation was posted, even if that period extends beyond the survey reporting period (or fiscal year). For program presentations made available via Facebook, count unique 1-minute views of each video. For those made available via other platforms, count unique views of each video OR PLAYS OF EACH AUDIO RECORDING. 

**Justification:**
Explicitly includes audio recordings, such as podcasts.

This revision and the proposed revision to 620 Total Number of Asynchronous Program Presentations (TOTPRES) should be considered together. 

**Potential methodological issues:**
Unknown. Also opens the idea of including listens/streams of live audio programs in 618 Synchronous Virtual Program Attendance (VIRATTEN). 

**States already collecting:**
Rhode Island

**Comment by yanademireva** [comment]
I would strongly urge against this change. More likely than not, survey respondents will interpret this to mean audio recordings that we count elsewhere, in audio downloadables. You can advise libraries to put podcast listens in there without formally changing the definition, and potentially compromising the data we get for audio downloadables.

**Comment by mariabernier10601** [comment]
Yana, have your libraries tended to mistakenly count their downloadable videos in this question in the past? Is that why you're worried they'll now also count downloadable audio?

I'm curious about what your comment implies about the libraries' confusion between library-created recorded video programs and circulating video/audio downloads.

**Comment by yanademireva** [comment]
@mariabernier10601 They have not, or at least I hope they haven't! :-( 

But, this could very well put us on the path to discussing whether these are "programs"/"program presentations," or collections (as other downloadable materials are). I believe this was touched on at some point when this element was introduced. Your comment made me think - what differentiates a library's self-produced podcast from a downloadable music album? I'm confident that I'll get this question, and I'm not sure I have a solid reason. 

Honestly, I think this data element is not particularly useful, at least for us in Maryland. Libraries produce content - it goes without saying. When we look back on the evolution of libraries from being repositories to being producers of content, is this the data point that will be key to our analysis? (Slight humor and sarcasm) Of course, that's not the point of this discussion, but I just wanted to throw it out there one more time. :-)

TL;DR - I'm concerned about where/why we would draw the distinction between this element, and other elements that count downloadables.

**Comment by enielsen-air** [comment]
I want to clarify that the potential confusion would be between "views/plays of program presentations" and "successful retrieval of electronic information" (as the latter currently exists). And there would only be confusion if the recordings were only accessible to authenticated library users. If they are freely available on from the library's website, YouTube channel, podcast feed, etc., then they shouldn't count as "retrieval" (again, as that definition currently exists).

Counting "total asynchronous presentations" as "audio units - downloadable" wouldn't be appropriate because the latter are only available to authenticated users AND have a set circulation period.

**Comment by KristenCooke** [comment]
I know we have a few libraries in our state with established podcasts, but we do not currently ask for this information.  The purposes of the podcasts can differ wildly, though. One of the larger producers uses their podcast as a means to update people about what is going on at the libraries, and that seems to be more of a promotional or PR initiative than it does a program. I think we could expect that other libraries use podcasts in this way. 

How do we avoid inadvertently tracking promotional efforts in the same arena as a Book Club podcast especially when the episodes could shift purposes dramatically? If the element moves ahead, it may address @yanademireva s concerns if we added verbiage indicating that it is audio content produced by the library staff or volunteers on behalf of the library.  I agree that these should be tracked, but I think the element could benefit from some extra exploration on the above topics.  

Given that there is a shift in libraries' production of information, I wonder if it would be more meaningful to move away from categorizing these recordings as "programming" and approach it from a lens of library-produced multimedia or original content. I think recorded content tells a different story than programming (both are valuable) and if it is expected to rise, we might benefit from looking ahead to long term trends.  Library-produced original content would encompass a few "stragglers" that are beneficial to measure, but may not currently have a home on the PLS.
